### ISSUE TITLE
dimensionality mismatch

### ISSUE THREAD

**Issue by vinayakumarr** [fernandoandreotti/cinc-challenge2017#4]
I ran ExtractFeatures() program. It created the following variable

allfeats_olap0.8_win10.mat

96229*173 allfeats.mat
8528*2 - references.mat
allfeats.mat file should contain 8528*173. But it contains 96229*173. Dimension mismatch between the feature samples and the labels. Could you please explain why?

**Comment by fernandoandreotti** [comment]
On [our paper](http://prucka.com/2017CinC/pdf/360-239.pdf) we describe this.

"To  account  for  the  varying  length  of  the  signals,  in  a second approach,  we divided the preprocessed ECG signals into 10-second segments with 50% overlap. First, we computed the features based on each segment (along each recording), and then computed the summary statistics such as mean standard deviation and min/max (for each feature), which were subsequently used in combination with bagged trees and neural network."

The variable `allfeats` contains the features for each segment and its length depend on the length of all individual recordings. Check the columns `rec_number` and `seg_number`.
